src/main/proto/phenopackets/schema/v2/core/genome.proto

```diff
@@ -33,90 +33,4 @@ message HtsFile {
   // A map of identifiers mapping an individual to a sample in the file. The key values must correspond to the
   // Individual::id for the individuals in the message, the values must map to the samples in the file.
   map<string, string> individual_to_sample_identifiers = 5;
-}
```

Comment:
Please keep these for the time being.

Reply:
I think we should restore the VCF class for use in a VRSATILE Extension for the cases where that is used, but otherwise all of these classes have direct analogues in VRSATILE. Instead of providing two options, can we try to converge on our approach to representing these?

@pnrobinson @larrybabb I would like your thoughts here as well.

Reply:
We talked this over and have decided it would be good to support this class natively as a VCFRecord. We'll make the corresponding changes to vrsatile.proto.

Reply:
Great. We really need this and would be a blocker for us not to have it. Keeping the original structure of the VCF record would be better than the proposed class I had. _i.e._ keep the `assembly, chr, pos, id, ref, alt, info` fields only.